.github/workflows/community.yml

```diff
@@ -0,0 +1,75 @@
+name: "community"
+on:
+  push:
+  pull_request_target:
```

Comment:
I understood quote the opposite when I read that the first time lol

in any case, the best solution is to avoid any secret at all for builds coming from forks